src/TestUtils/src/UITest.Appium/Actions/AppiumLifecycleActions.cs

```diff
@@ -61,12 +61,6 @@ CommandResponse LaunchApp(IDictionary<string, object> parameters)
 					{ "bundleId", _app.GetAppId() },
 				});
 			}
-			else if (_app.GetTestDevice() == TestDevice.Windows)
-			{
-#pragma warning disable CS0618 // Type or member is obsolete
-				_app.Driver.LaunchApp();
```

Comment:
`LaunchApp ` have been removed.

Cannot execute the launchApp script because not exists.

`_app.Driver.ExecuteScript("windows: launchApp", new Dictionary<string, object>());`

And `ActivateApp `is not implemented.
```
if(_app.Driver is WindowsDriver windowsDriver)
     windowsDriver.ActivateApp(_app.GetAppId());
```

The alternative could be to quit the session and create a new one, but it is a bigger change. The Windows tests are going to fail until fix this.